src/libraries/Microsoft.Extensions.Configuration.Binder/gen/ConfigurationBindingGenerator.cs

```diff
@@ -73,7 +71,7 @@ private sealed record CompilationData
 
             public CompilationData(CSharpCompilation compilation)
             {
-                LanguageVersionIsSupported = compilation.LanguageVersion >= LanguageVersion.CSharp11;
+                LanguageVersionIsSupported = compilation.LanguageVersion >= LanguageVersion.Preview;
```

Comment:
Presumably to require interceptor support and since there's no `CSharp12` value available yet? I suspect this check would be broken for later releases of Roslyn, so perhaps we need to [polyfill the value for `CSharp12`](https://github.com/dotnet/roslyn/blob/168689931cb4e3150641ec2fb188a64ce4b3b790/src/Compilers/CSharp/Portable/LanguageVersion.cs#L218-L232) and take that into account?